heuristic/alns.py

```diff
@@ -377,8 +461,7 @@ def iterate(self, iterations=None, runtime=None):
         logger.error(f"Initial solution: {self.initial_solution.get_objective_value(): .2f}")
         logger.error(f"Best legal solution: {self.best_legal_solution.get_objective_value(): .2f}")
         logger.error(f"Best solution: {self.best_solution.get_objective_value(): .2f}")
-
-        candidate_solution.write("solutions/heuristic_solution_2")
+        #self.best_solution.write("solutions/best_solution")
```

Comment:
I think it would be good to also save the best (or best_legal) solution after each run now. I would prefer to add the same naming scheme I have implemented in PR #168, that is `CURRENT_TIME-name-ALNS.sol`. `name` defaults to `PROBLEM-mode=MODE` but could be defined by the user. Thoughts?

Reply:
When running the final tests I believe we should only save the best legal solution. No reason to save anything else then. Until then I like saving the answers in different places to see if the operators work as expected or look for improvements